src/herder/HerderSCPDriver.cpp

```diff
@@ -351,7 +347,13 @@ HerderSCPDriver::validateValueHelper(uint64_t slotIndex, StellarValue const& b,
 
         res = SCPDriver::kInvalidValue;
     }
-    else if (!txSet->checkValid(mApp, 0))
+    else if (!txSet->checkValid(mApp,
+                                curProtocolPreservesTxSetCloseTimeAffinity()
```

Comment:
those two parameters are equal, we should use a variable instead of duplicating

Reply:
Agreed and changed.